azure-pipelines.yml

```diff
@@ -6,6 +6,7 @@ trigger:
 jobs:
 
 - job: Windows
+  condition: not(startsWith(variables['Build.SourceBranchName'], 'azure-linux-'))
```

Comment:
What's up with this branch here and in MacOS?

Reply:
It's a "feature" I added to make it easier to fix the linux build.
* Any branch starting "azure-" is not built on AppVeyor now.
* Any branch starting "azure-linux-" only runs the linux job on Azure.

This speeds things up quite a bit when working on the scripts themselves. I should probably finish it up by adding similar logic for "azure-windows-" and "azure-macOS-" but I only put in what I needed immediately.

BTW I tried to do it by setting variables at the start of the script, before the jobs, and then referring to them in each job. But the subtleties of how static vs dynamic variables are defined in the yml made it too much trouble, so I used copy and paste. If you know of an example somewhere, that would be helpful.

Reply:
I set it up to use `azure-windows-` and  `azure-macOS-` prefixes the same as `azure-linux-`. Ran tests of all three to verify they work.